pkg/azurefile/controllerserver.go

```diff
@@ -132,12 +133,12 @@ func (d *Driver) DeleteVolume(ctx context.Context, req *csi.DeleteVolumeRequest)
 
 	accountKey, err := GetStorageAccesskey(d.cloud, accountName, resourceGroupName)
 	if err != nil {
-		return nil, fmt.Errorf("no key for storage account(%s) under resource group(%s), err %v", accountName, resourceGroupName, err)
+		return nil, status.Errorf(codes.Internal, "could not get the storage access key: %v", err)
 	}
 
 	klog.V(2).Infof("deleting azure file(%s) rg(%s) account(%s) volumeID(%s)", fileShareName, resourceGroupName, accountName, volumeID)
 	if err := d.cloud.DeleteFileShare(accountName, accountKey, fileShareName); err != nil {
-		return nil, err
+		return nil, status.Errorf(codes.Internal, "could not delete the volume: %v", err)
```

Comment:
`return nil, status.Errorf(codes.Internal, fmt.Errorf("DeleteFileShare %s under %s failed with error: %v", fileShareName, accountName, err)`

Reply:
I think here is `return nil, status.Errorf(codes.Internal, "DeleteFileShare %s under %s failed with error: %v", fileShareName, accountName, err)`.